src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/PostgresDatabaseHandler.cs

```diff
@@ -0,0 +1,31 @@
+using Altinn.Platform.Events.Configuration;
+using Npgsql;
+using Npgsql.Logging;
+
+namespace Altinn.Platform.Events.Repository
+{
+    /// <summary>
+    /// Handling the postgres db
+    /// </summary>
+    public class PostgresDatabaseHandler
+    {
+        private readonly PostgreSQLSettings postgresSettings;
```

Comment:
consider using _postgresSettings to match notation of private vars in the other projects.

Reply:
Removed the class, moved the logic to PostgresRepository